plugins/channel_header_plug/channel_header_plug.jsx

```diff
@@ -133,8 +135,28 @@ export default class ChannelHeaderPlug extends React.PureComponent {
         );
     }
 
-    createDropdown = (plugs) => {
-        const items = plugs.map((plug) => {
+    createActionButton = (plug) => {
```

Comment:
Should this `plug` variable be something like `plugAction` instead? Right now it reads as "One plugin" instead of "One item that came from a plugin"

Reply:
I was being coherent on the previous implementation, but you are right.